client/src/browse/list/ItemList.js

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect, useState} from "react";
+import styled from "styled-components";
+
+import ItemCard from "./ItemCard.js";
+
+const ItemList = (props) => {
+
```

Comment:
@MarkAlperin Im sure you know what Im going to say already lol

```
const ItemList = ({ items, selectedItem, setSelectedItem }) => {

```
As a thanks I believe that should get all the props